static/js/dashboard/controllers/dashboard.controller.js

```diff
@@ -23,6 +23,9 @@
       self.getSavedTask = getSavedTask;
       self.dropSavedTasks = dropSavedTasks;
 
+      getWorkerData();
```

Comment:
@dmorina getWorkerData() is actually not of any use in the dashboard, as it only concerns with requester ranking. I copy-pasted this code of @nistala as it is, just for the sake of preserving it. It can be removed from the dashboard controller, services etc